topology/probes/ovs_of.go

```diff
@@ -292,29 +292,35 @@ func launchOnSwitch(cmd []string) (string, error) {
 	return "", err
 }
 
-// launchContinuousOnSwitch launches  a stream producing command on a given switch
+// launchContinuousOnSwitch launches  a stream producing command on a given switch. The command is resilient
+// and is relaunched until the context explicitely cancels it.
 func launchContinuousOnSwitch(ctx context.Context, cmd []string) (<-chan string, error) {
 	var cout = make(chan string, 10)
-	out, err := executor.ExecCommandPipe(ctx, cmd[0], cmd[1:]...)
-	if err != nil {
-		close(cout)
-		return cout, err
-	}
-	reader := bufio.NewReader(out)
-	var line string
 
 	go func() {
-		for {
-			line, err = reader.ReadString('\n')
-			if err == io.EOF {
+		for ctx.Err() == nil {
+			out, err := executor.ExecCommandPipe(ctx, cmd[0], cmd[1:]...)
+			if err != nil {
```

Comment:
log the error here like
logging.GetLogger().Errorf("Can't execute command %v", cmd)

Reply:
Yes missing logging.